examples/tensorflow/simple.yaml

```diff
@@ -12,7 +12,8 @@ spec:
         spec:
           containers:
             - name: tensorflow
-              image: kubeflow/tf-mnist-with-summaries:latest
+              image: docker.io/kubeflowkatib/tf-mnist-with-summaries:latest
```

Comment:
Where is the source Dockerfile for this image?

Reply:
Here: https://github.com/kubeflow/katib/tree/888bec38f4d8d6b43a0606eb721809ec045be113/examples/v1beta1/trial-images/tf-mnist-with-summaries